### DISCUSSION TITLE
Feature Request - Disable Mandatory Customer Selection on POS Devices

### DISCUSSION THREAD

**Discussion by danielsonfire** [Blair2004/NexoPOS#1384] [General]
**As a frequent shopper, I have noticed that many point of sale (POS) systems in retail stores require customers to select their name or enter their information every time they make a purchase. While this may be useful for collecting data on customer behavior, it can be an inconvenience for customers who simply want to make a quick purchase.

I am requesting that the mandatory customer selection feature on POS devices be disabled, allowing customers to complete their transactions more quickly and efficiently. This feature could be replaced with an optional customer selection feature that would allow customers to input their information if they choose to do so.

I understand that some retailers may find customer data collection to be a valuable tool for marketing and customer retention, but I believe that making this feature optional would still allow retailers to collect data from willing customers while also improving the checkout experience for those who prefer to skip the selection process.

Thank you for considering my request.

**Comment by josephelkhoury** [comment]
Hello,

It's true that for restaurants for example, we don't usually need the customers information.
What we did as a workaround, is that we created a customer named "N/A" and set it as the default customer. 
So for each order, this customer will be selected by default.

**Reply by danielsonfire** [reply]
Everytime we need to select customer right before checking out

**Reply by josephelkhoury** [reply]
No no, you can set it as a default customer in Settings -> Customers -> Default Customer Account.

<img width="1455" alt="Screenshot 2023-03-20 at 9 24 13 AM" src="https://user-images.githubusercontent.com/1782788/226273010-47ab12bf-98b0-45b9-bc2e-2ccc7c0a8333.png">
